app/models/comment.rb

```diff
@@ -0,0 +1,18 @@
+class Comment < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  # Attributes
+  attribute :text, :text
+
+  # Callback
+  after_save :update_post_comments_counter
+
+  private
+
+  # Methods
+  def update_post_comments_counter
+    post.update(comments_counter: post.comments.count + 1)
+  end
```

Comment:
This method has the same issue that the `update_user_posts_counter` has. It can also be fixed by removing the + 1 on line 16: 
```
def update_post_comments_counter
    post.update(comments_counter: post.comments.count)
end
```